package.json

```diff
@@ -42,7 +42,7 @@
     "loopback": "^2.22.0",
     "loopback-boot": "^2.6.5",
     "loopback-component-explorer": "^2.1.0",
-    "loopback-datasource-juggler": "^2.39.0",
+    "loopback-datasource-juggler": "^3.0.0",
```

Comment:
I am afraid you are changing a wrong place. You should remove this line entirely and use `loopback: '^3.0.0'` instead, see L42.